docker-compose.yml

```diff
@@ -5,16 +5,16 @@ services:
     user: vscode
     # add groups from host for render, plugdev, video
     group_add:
-      - 109 # render
-      - 44  # video
-      - 46  # plugdev
+      - "109" # render
+      - "44"  # video
+      - "46"  # plugdev
     shm_size: "256mb"
     build:
       context: .
       dockerfile: docker/Dockerfile.dev
-    devices:
-      - /dev/bus/usb:/dev/bus/usb
-      - /dev/dri:/dev/dri # for intel hwaccel, needs to be updated for your hardware
+    #devices:
```

Comment:
Did you have to comment these out for it to work? Are you running the latest docker and docker-compose versions?

Reply:
Oh sorry didn't mean to include that in the PR. But yeah I did have to, I am running Docker Hub on Mac so I will check for update and see.